runtime/onert/test/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ target_link_libraries(${TEST_ONERT} onert_core)
 target_link_libraries(${TEST_ONERT} gtest)
 target_link_libraries(${TEST_ONERT} gtest_main)
 target_link_libraries(${TEST_ONERT} ${LIB_PTHREAD} dl)
+
+# Requires linking nnfw_coverage: check header coverage
+target_link_libraries(${TEST_ONERT} nnfw_coverage)
```

Comment:
(out of curiosity) Do we need this line for all onert tests?

Reply:
Maybe we need...